Writing edits for 1.39

Ven-edits opened the pull request:
Hope I did this right. Made a slight tweak so the intended file is the one titled Hermione Intro Dialogue Edits. Please disregard Dialogue edits on Hermione Intro if that comes through too.

TropeCode commented:
I'm not sure which changes you intended to submit, but you can see what is part of this pull request under the "Files changed" tab. If I understand you correctly, the only change is the "stil stil" correction in the intro, not the other stuff?

A cursory glance of the current changes tells me there are a lot of edits that we wouldn't necessarily accept, because they don't resolve spelling or grammar issues but actually change the meaning and/or flow of the dialogue.

Ven-edits commented:
Ah, great. That's useful to know.

I did say to Johnny that I'll do a deep edit on that file and if he wants
me to tone things back and focus on specific things I can cater to that.
None of the edits have changed the core dialogue that I'm aware of. I'd say
80% are tweaks to punctuation--toning back unnecessary ellipsis and
exclamation marks to make the remaining ones more impactful, adding missing
full stops, correcting capitalisation (though I'm not even sure that's
noticeable in game), playing around with titles like Professor/Sir to add
the nuance of their use, or tightening up word combos like 'we will, I
have' or It is into the more natural speech of 'We'll/I've/It's'.
There's a couple of lines where I've suggested a rewording of a sentence
for slightly better grammar, but the core of the sentence is the same.
I did unite one of Snape's (I think) lines which was fractured into two
parts. Maybe you want to keep that split for some perceived impact, but I
didn't see how it added anything other than an extra click/wait for the
player.
But beyond that I didn't want to change any of the actual text for the
reasons you laid out. However there were two places where it looks like
there's inconsistencies in the dialogue. Without knowing how to draw
attention to it, I left # notes beside them explaining the issue and if
you, Johnny, or whoever think there's an issue there then you can fix it.
One is line 786-789 where in the space of 3 sentences Hermione says she has
to go and do two different things. Then at 911 Genie asks a question in one
of his optional responses that Hermione doesn't appear to ever acknowledge
or answer.
I think I also questioned the super long lines of ............... and
whether you wanted to decide on a set amount for consistency or just knock
them down to the standard three. But those are leftover from Akabur I
believe. He has a thing for face mashing the full stop key and just just
seeing where it lands. I think one time they went right out of the text
window, which, I have to admit, made me laugh pretty hard. As I said to
Johnny though, I don't think dodgy punctuation is as distracting to players
as much as typos or awkward sentences. For the most part you just zone
punctuation out once you get the feel for what the writer's quirks are.
Whereas typos or strange sentence construction sticks out like a sore thumb
and throws you out of the immersion... which I guess is an argument not to
care about all those punctuation edits.

But yeah, at the end of the day they're all intended as suggestions. I'm
still not 100% sure how this works, but I presumed you guys would scroll
through and confirm/reject what you see fit. In a brief scan I see I fat
fingered a # instead of a ' on line 1338 -_-, so that would need fixing
before accepting <sigh>. But if it's a case that the file has to be picture
perfect and you either accept all or nothing then I can go back and just do
a basic proofread and pick out the handful of odd typos and missing
punctuation. It all depends on how much use you guys find from it at the
end of the day and what your overall vision for the game is.

David.

On Thu, Aug 20, 2020 at 12:35 AM TropeCode <notifications@github.com> wrote:

> I'm not sure which changes you intended to submit, but you can see what is
> part of this pull request under the "Files changed" tab. If I understand
> you correctly, the only change is the "stil stil" correction in the intro,
> not the other stuff?
>
> A cursory glance of the current changes tells me there are a lot of edits
> that we wouldn't necessarily accept, because they don't resolve spelling or
> grammar issues but actually change the meaning and/or flow of the dialogue.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/SilverStudioGames/WT-Silver/pull/20#issuecomment-676811370>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQT7ZTCLBA67VW6EICLI3BTSBROUXANCNFSM4QEKNAQQ>
> .
>

TropeCode commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -139,73 +139,73 @@ label hermione_intro_E1:
+        call nar(">You keep stroking your cock.")
     else:
-        m "*Ehm*... I suppose you did. I mean, we did."
+        m "*Ehm*... I suppose you did. I mean we did."
```
And keep the comma.

Ven-edits replied:
Take into account that commas and full stops also create pauses in reading.
So there are three ways you can imagine him saying the second sentence:
- He could do it in one fluid motion having realised his mistake, in which case we'd just remove the comma.
- He can make an audible pause BEFORE the 'we', likely because he's double checking himself to ensure he uses the correct word this time around. (I'd use an ellipsis for this, but a comma technically works too.)
- Or there's an audible pause AFTER the word 'we', because he's putting extra stress on it acknowledging that's what he ballsed up. For that you'd put the 'we' in italics but remove the comma.

Bold does appear in videogame text from time to time, but it's usually to highlight important need to know details for players who skim read so they notice thinkgs like place names or directions for what they're supposed to do next.

For now I've put it in italics and left the comma out because I think that's what you were going for. If you want me to also put the comma back as well I can, but in my ears I don't hear a break before the 'we' to justify a comma or ellipsis here.

TropeCode commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -411,13 +410,13 @@ label hermione_intro_E2:
                 "\"Absolutely not! I'm busy! Come back later!\"":
                     her "But..."
-                    her "Alright... I will come back tomorrow then..."
+                    her "Alright. I'll come back tomorrow then."
```
I would actually keep the first pause (as well)

TropeCode commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -450,73 +449,73 @@ label hermione_intro_E2:
+            call her_main("...", "annoyed", "squint", "angry", "mid")
+
+    her "I see that no matter what I do I simply cannot get through to you, Sir."
+    call her_main("So, in light of your negligence, I've decided to take the initiative myself.", "open", "closed", "angry", "mid")
```
Commas are fine here (especially with her sounding formal), just the exclamation mark at the end is important.

TropeCode commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -538,66 +537,66 @@ label hermione_intro_E2:
-    call sna_main("But all I did was give her more leverage to harass me with...","snape_43")
-    call sna_main("She's even turning the teachers against me now...","snape_43")
+    call sna_main("Yeah, that word works.","snape_31")
+    call sna_main("But all I've done is give her more leverage to harass me with.","snape_43")
```
Not sure if the change in tense is better either.

TropeCode commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -538,66 +537,66 @@ label hermione_intro_E2:
+    call sna_main("She's never failed a single test in her entire time here.","snape_31")
+    call sna_main("And she's always ahead of the schedule.","snape_31")
+    call sna_main("Fuck, how I hate it when she corrects me during my classes.","snape_08")
+    call sna_main("And thanks to her the Gryffindor house's points are way ahead of everybody else now.","snape_34")
```
I feel like "house points" are implied here (_totum pro parte_), so the original works better.

TropeCode commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -696,99 +695,99 @@ label hermione_intro_E3:
-            call her_main("You can stare at me all you want, sir.", "normal", "squint", "angry", "mid")
-            her "It will not make the problems of this school go away."
+            call her_main("You can stare at me all you want, Sir.", "normal", "squint", "angry", "mid")
+            her "It won't make the problems of this school go away."
```
No contraction. She's trying to sound articulate.

TropeCode commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -696,99 +695,99 @@ label hermione_intro_E3:
-            call her_main("The best tutor is a book, and I have the entire Hogwarts library at my disposal.", "open", "closed", "base", "mid")
-            call her_main("I don't think it would be necessary, sir. But...", "soft", "base", "base", "mid")
+            call her_main("Yes, that might help I suppose.", "soft", "base", "base", "R")
+            call her_main("But while I appreciate your offer, Sir...", "open", "base", "base", "mid")
```
I'd keep the original sentence and the pause on the line before, because it flows better into her next line

TropeCode commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -959,32 +958,32 @@ label hermione_intro_E5:
-    call ton_main("Hearing those naughty words spill out of her gorgeous little mouth really gets me going!", "soft", "narrow", "base", "R", hair="horny")
-    g9 "I can imagine so."
+    call ton_main("To persuade her to trade favours herself, for a start.","open","base","base","R")
+    call ton_main("Convince her that it isn't all bad.","base","base","base","mid")
```
I think the point of this edit was to have her mention favour trading first, in the sentence before, so that "it" can be used to refer to that in the second sentence. Otherwise it's not clear what "it" refers to in the first sentence.

Ven-edits replied:
As Trope said. In the original version she's discussing 'it' in the first sentence before favour trading has been mentioned, so the sentence. Genie did ask her what she was suggesting, so it makes sense and fits her character to be forthright with him.

I could change it to "To persuade her to trade points for favours for a start." if it's sounding funny and you want more clarity.

TropeCode commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -959,32 +958,32 @@ label hermione_intro_E5:
-    call ton_main("How could I possibly get tired of that!", "crooked_smile", "narrow", "base", "stare")
-    call ton_main("I'm very glad I decided to join you two.","open","base","base","down")
-    call ton_main("As an Auror It's just constant busy work...", "open", "base", "raised", "mid")
+    call ton_main("I'll never get tired of that!", "crooked_smile", "narrow", "base", "stare")
```
Keep the original sentence. It's stated as a rhetoric question for emphasis.

TropeCode commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -1304,23 +1303,23 @@ label hermione_intro_E6:
+            her "We really need those points."
+            m "..."
+            call her_main("Of all the esteemed wizards and witches in this school...", "annoyed", "base", "worried", "R")
+            her "You're the only one I don't mind asking favours from."
```
The original two sentences  work better and have a somewhat different meaning.

Ven-edits replied:
"You're the only one I don't mind doing favours for."?

In response to Trope: I thought the original sentence was off because it sounds like she's suggesting that being an esteemed wizard is a rare thing. Are there any wizards who can give out points who aren't esteemed? Snape even calls himself an esteemed wizard on line 545.
To me, if the crux of what she's saying is 'you're the only one I trust here', then the opening sentence should be acknowledging the entire roster of teachers. You could remove the witches if the favours are accepted as only being sexual in nature, but I assumed there was all sorts of favour trading going on, which is somewhat proven when he's dishing out points for her doing work, or telling him about her day, or sticking out her tongue.

TropeCode commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -1329,22 +1328,22 @@ label hermione_intro_E6:
+        call her_main("{size=-4}(Students show teachers their tongues all the time.){/size}", "soft", "base", "base", "R")
+        call her_main("{size=-4}(Although that's usually when the teacher isn't looking.){/size}", "base", "narrow", "base", "R_soft")
+        call her_main("{size=-4}(But there's nothing wrong with what I did today.){/size}", "annoyed", "base", "base", "L")
+        call her_main("{size=-4}(I earned Gryffindor extra points.){/size}", "smile", "happyCl", "base", "mid")
```
Also keep "my house", it makes it sound more personal to her.

Ven-edits commented:
Hi Johnny,
Thanks for taking the time to consider.

Things like the Professor and Sir was me probably taking a few liberties.
Generally speaking it's weird to switch between them and Professor is an
uglier word to say all the time. I wonder though, what do they call
Dumbledore in the books? The obvious answer here is to just have Hermione
call him whatever she usually does, because the name that she eventually
settles on is the one the player ultimately chooses.

The *Hmm*... was also a bit of a weird one. I did that for consistency
because everywhere else whenever they make a throaty sound like tsk or hmpf
or erm/ehm it's inside stars and with ellipsis after it. I think that was
probably the original Akabur writing, but were it me I'd have just removed
the stars from the others but it seemed simpler to just make the odd Hmm's
that got away consistent if that was the intended way to portray those
sounds.

So what's the process now? Do I go back to my file and accept/reject the
ones you want and resubmit a pull request? Or is this just a feedback for
me to see what you want in the future?

David.

On Tue, Aug 25, 2020 at 6:56 PM TropeCode <notifications@github.com> wrote:

> *@TropeCode* commented on this pull request.
> ------------------------------
>
> In game/_script_/_hermione_/events/_intro_.rpy
> <https://github.com/SilverStudioGames/WT-Silver/pull/20#discussion_r476635260>
> :
>
> > @@ -1329,22 +1328,22 @@ label hermione_intro_E6:
>      # Hermione inner thoughts
>      if current_favor == "show_tongue":
>          call her_main("{size=-4}(*Hmm*...){/size}", "annoyed", "narrow", "worried", "down", xpos="base", flip=True, trans=d3)
> -        call her_main("{size=-4}(Students show teachers their tongues all the time...){/size}", "soft", "base", "base", "R")
> -        call her_main("{size=-4}(Although that's usually when the teacher is not looking...){/size}", "base", "narrow", "base", "R_soft")
> -        call her_main("{size=-4}(But there is nothing wrong with what I did today...){/size}", "annoyed", "base", "base", "L")
> -        call her_main("{size=-4}(I earned my house extra points...){/size}", "smile", "happyCl", "base", "mid")
> +        call her_main("{size=-4}(Students show teachers their tongues all the time.){/size}", "soft", "base", "base", "R")
> +        call her_main("{size=-4}(Although that's usually when the teacher isn't looking.){/size}", "base", "narrow", "base", "R_soft")
> +        call her_main("{size=-4}(But there's nothing wrong with what I did today.){/size}", "annoyed", "base", "base", "L")
> +        call her_main("{size=-4}(I earned Gryffindor extra points.){/size}", "smile", "happyCl", "base", "mid")
>
> Also keep "my house", it makes it sound more personal to her.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/SilverStudioGames/WT-Silver/pull/20#discussion_r476635260>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQT7ZTDIOOW7EHNB6SLJSQLSCP3LHANCNFSM4QEKNAQQ>
> .
>

TropeCode commented:
If you make additional changes to your current files, they should be automatically included in this pull request. So you can make the suggested changes and then we can have another look.

Ven-edits commented:
I've made all the changes you've requested to my file. Any that came out
slightly different or you weren't certain about I've copied below with
comments on what I did.There's only 13 now!
I assume I just do a new pull request from my files and it will update the
one that's already there on the server? And if so, now that the update's
out, is it still going to the 1.39 branch or is there a new one to send to?

David.

In game/_script_/_hermione_/events/_intro_.rpy
<https://github.com/SilverStudioGames/WT-Silver/pull/20#discussion_r475898047>
:

>      else:
-        m "*Ehm*... I suppose you did. I mean, we did."
+        m "*Ehm*... I suppose you did. I mean we did."

This looks a bit weird, perhaps we should bold "we"
Trope -- And keep the comma.

Take into account that commas and full stops also create pauses in reading.
So there are three ways you can imagine him saying the second sentence. He
either says it in one fluid motion in a panic because he realises his
mistake, in which case you'd just remove the comma. Or he can make an
audible pause BEFORE the 'we', because he's double checking himself to make
sure he uses the correct word this time around. (Personally, I'd use an
ellipsis for that, but a comma works too.) Or you can hear an audible pause
AFTER the word 'we', because he's putting extra stress on that word since
that's the replacement to the word 'you' that he wants to overwrite. In
that case you'd put it in italics. Bold does appear in videogame text from
time to time, but it's usually to highlight important need to know details
for the benefit of players who skim read such as place names or directions
for what they're supposed to do next.

For now I've put it in italics and left the comma out because I think
that's what you're going for. If you want me to also put the comma back as
well I can, but in my ears I don't hear a break before the 'we' here
because this is supposed to be a quick correction on his part. The comma
does work if he resays the entire sentence though, but I don't think that's
necessary.


In game/_script_/_hermione_/events/_intro_.rpy
<https://github.com/SilverStudioGames/WT-Silver/pull/20#discussion_r475908964>
:

>
         call gen_chibi("cum_behind_desk_done")
         with d3
         pause.5

-        m "Yes, girl. It's all exactly as you say and I will.... take
care of it all."
+        m "Yes, girl. It's all exactly as you say and I will... take
care of it."

I'd keep "all" Cause genie is referring to something different than
Hermione. It doesn't sound right in context to what Hermione said which is
sort of the point.
I removed the all here because it's doubling up from the one at the front
of the sentence. He's already made 'all that she says' the topic so there's
no need to repeat it at the end. You could remove the first all and leave
the second one in though, but to my ears that doesn't sound as natural as
leaving the first in.

+    m "Never mind. I'm a genie, remember? I know things."
+    call sna_main("Hmm... And yet you still need me to teach you
magic?","snape_37")

"*Hmm*"
I assumed you wanted the stars and not italics for this so that's what I've
done.

In game/_script_/_hermione_/events/_intro_.rpy
<https://github.com/SilverStudioGames/WT-Silver/pull/20#discussion_r475924254>
:

-    m "Well, alright, let it out..."
-    call sna_main("How come one tiny... cunt, is able to cause me so
much grief?!","snape_31")
+    m "Well, alright, let it out."
+    call sna_main("How come one tiny... cunt is able to cause me so
much grief?","snape_31")

This doesn't look right to me. The pacing seemed better before.
Yeah, I pondered on this one a fair bit too. The comma isn't correct
because it's splitting the sentence in two, and right off another pause at
that. I'm wondering if what you were going for was emphasis on the word
cunt like with the 'we' earlier on, in which case it would be in italics
and the comma removed. For now I've put the comma back in, but it doesn't
sit right with me.

-    call her_main("How would this make me look?", "angry", "base",
"base", "mid", tears="soft")
+    call her_main("And the worst part is that I think I might be the
only one who failed.", "angry", "base", "base", "mid", tears="soft")
+    call her_main("How would that make me look?", "angry", "base",
"base", "mid", tears="soft")

Does this need changing? "How will this make her look like in the eyes of
the other students" is what she's saying.
The word 'this' feels wrong to me. 'This' is more immediate than 'That', as
if the event has either happened and she's looking at the repercussions, or
that there's an object visually present that's being referred to in order
to support her statement. Because she's only hypothesising that she may be
the only one who failed, and it's not something that's happened or can be
seen, the word 'that' feels more appropriate here so I've left it in for
now. But it's your call.

+    call her_main("Well, I'd better get going. We have another
\"MRM\" meeting today.", "open", "closed", "angry", "mid")
+    her "I'll let you know about the new ideas we come up with."
+    m "I can hardly wait."
+    call her_main("Well, if there's nothing else, I have a studying
schedule to keep.", "open", "closed", "base", "mid") # this is
potentially doubling up her reason to leave. She's just said she has
an MRM meeting to go to, so you could consider changing the line.

I think it's fine. It's a bit weird as it doubles up as you said. But she's
trying really hard to seem more important than she actually is. Not really
worth changing.

In light of your comment about her wanting to seem important/organised I've
tweaked this slightly and brought the study schedule reference up behind
the MRM meeting, which has also eliminated the need for Genie's second
response. I can change it back if you want, but this did stand out to me as
an error when I was playing the game. The issue is that the two sentences
have the same construction and are delivering the same information with
different details, right down to the 'Well,' at the front. So from my
player perspective it looked like one was created to update and overwrite
the other, but the original got left in by mistake.

"\"Splendid! Starting today?\"": # I'm unsure how these chat options
connect just yet, but this is Genie speaking, right? He's asking her a
question, but when Hermione comes back in she follows up by saying
'let me know when' rather than answering his question, so you might
want to leave it at Splendid!

When it has a "pass" under it it usually means that it just skips to the
next chunk of text and the dialogue choice is flavour writing.

Fair enough. Though as a player I assumed that the multiple choices were
what I was choosing for Genie to say/think/do. So when there's a question
like 'Starting today?' in the choice box I'd be expecting Hermione to
answer it. And she sort of does when she says 'Let me know when and I'll
bring my books', but then she just walks out leaving me wondering why Genie
never bothered to give her the time to come back at. But I've left it as it
was for now.

+    call her_main("But that's not all.", "normal", "squint", "angry", "mid")
+    her "The \"MRM\" shall have to investigate our education system
much closer."

Don't change this sentence.

I've returned the now and the ellipsis on the end, but I've left 'have' in
there. It's correct grammar to do so because she's saying that this change
is coming as a result of her failing the test, whereas without the have it
suggests this is something she's doing on Genie's behalf as a result of
some event and she's keeping him in the loop. Unless she was meant to
be making this MRM change on his behalf? But the MRM seems to be her pet
project and Genie's already made it clear that he thinks it's ridiculous so
that comes off odd to me.

In game/_script_/_hermione_/events/_intro_.rpy
<https://github.com/SilverStudioGames/WT-Silver/pull/20#discussion_r475964863>
:

>                      $ d_flag_01 = "desk"
                     $ d_flag_02 += 1
-                    her "*uhm*... alright..."
+                    her "*Uhm*... Okay."

I'd have it say "alright"... here. It's not as British sounding but it
sounds more "deafeat-ish"
I switched Hermione's alrights to okays because alright is used quite a lot
as a default acknowledgement. Since Genie and Hermione are together so
often, I thought giving her okay helps to differentiate her voice a bit
from his so that they're not using the same talking quirks. I've left it as
okay for now, but I can change this back to alright if you want. But they
mean the exact same thing, so I wouldn't read alright as coming across any
more defeated than okay.

In game/_script_/_hermione_/events/_intro_.rpy
<https://github.com/SilverStudioGames/WT-Silver/pull/20#discussion_r475970330>
:

>                  m "How badly do you need those points?"
                 call her_main("..................", "annoyed",
"squint", "angry", "mid")
-                call her_main("Yes, sir. I do need to get spanked.",
"open", "closed", "angry", "mid")
-                m "Alright, that's good enough for now..."
+                call her_main("Yes, Sir. I deserve to be spanked.",
"open", "closed", "angry", "mid")

Don't change this sentence. This is early on and she says that she needs to
get spanked because she needs the points. You've missed the point here
somewhat.
I did miss the point here it seems. But to me, saying need suggests that
she wants to get spanked which clearly isn't the case, so I changed it to
deserve as that subtly satisfies his demand but isn't explicitly asking for
it. I've switched it back, but you could consider putting the italics
stress on the word 'need' to show that she's playing on the word he just
said and making it about the points to remove the suggestion that
it's something she wants.

>

-            call her_main("You are an esteemed wizard and to be
honest...", "annoyed", "base", "worried", "R")
-            her "The only person in this school whom I don't mind
asking for this..."
+            call her_main("Of all the esteemed wizards and witches in
this school...", "annoyed", "base", "worried", "R")
+            her "You're the only one I don't mind asking favours from."

She is the one selling favours. "You're the only one I don't mind asking
for something like this..." Would work better.

Trope -- The original two sentences work better and have a somewhat
different meaning.

"You're the only one I don't mind doing favours for."?
In response to Trope: I thought the original sentence was off because it
sounds like she's suggesting that being an esteemed wizard is a rare thing.
Are there any wizards who aren't esteemed? I suppose if she sees the rest
of the teachers like Snape there's some logic to it. But I thought even
Snape was considered to be a competent spell caster. To me it just seems
that if she's playing the 'you're the only one I trust here' line, then the
opening sentence should be acknowledging the entire roster of teachers.

In game/_script_/_hermione_/events/_intro_.rpy
<https://github.com/SilverStudioGames/WT-Silver/pull/20#discussion_r475974114>
:

>              pass

-    call her_main("Thank you, professor. Thank you so much!",
"smile", "happyCl", "base", "mid")
-    call her_main("Well... I suppose I'd better go now...", "base",
"base", "base", "mid")
+    call her_main("Thank you, Professor Dumbledore. Thank you so
much!", "smile", "happyCl", "base", "mid")

Students rarely refer to the teacher by their name, it's always "professor"
or "sir".

I changed it back, but I made this tweak because it's more personal and
emotional, while at the same time it reminds the player that she's seeing
him as Dumbledore and not Genie. Regarding your point, it might be
different between schools, but when I went to school we'd never refer to a
teacher by their job title alone--partly why I changed so many Professors
to Sirs. It would always be Professor/Mr/Mrs/Miss <surname> when being
formal, or Sir/Ma'am/Miss for casual conversation where you need to refer
to them to get their attention. In general, to use the job title is a very
formal means of communication--though there's always the one twat with
their head so far up their own arse that they demand it be used all the
time to remind you of their status.

-        call her_main("{size=-4}(I am a very bad girl...){/size}",
"base", "base", "angry", "stare")
-        call her_main("{size=-4}(Yes, I can say things like that
easily...){/size}", "smile", "happyCl", "base", "mid")
+        call her_main("{size=-4}(I'm a very bad girl.){/size}",
"base", "base", "angry", "stare")
+        call her_main("{size=-4}(Yes, it's easy to say meaningless
things like that.){/size}", "smile", "happyCl", "base", "mid")

Don't change this sentence.

I've put it back, but I've moved easily up front since having it on the end
is a strange construction to have the adverb on the end of a pronoun, which
is what originally drew my eye to it. So it now reads "Yes, I can easily
say things like that..."

-    call ton_main("To persuade her into having a try of it herself,
for a start...","open","base","base","R")
-    call ton_main("Convince her that trading favours isn't all
bad.","base","base","base","mid")
+    call ton_main("To persuade her to trade favours herself, for a
start.","open","base","base","R")
+    call ton_main("Convince her that it isn't all
bad.","base","base","base","mid")

I've left this change alone for now because, as Trope said, in the
original version she's discussing 'it' in the first sentence before
she's determined what 'it' is. I could change it to "persuade her to
trade points for favours" if it's sounding funny and you want more
clarity.


On Wed, Aug 26, 2020 at 11:30 PM TropeCode <notifications@github.com> wrote:

> If you make additional changes to your current files, they should be
> automatically included in this pull request. So you can make the suggested
> changes and then we can have another look.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/SilverStudioGames/WT-Silver/pull/20#issuecomment-681156750>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQT7ZTG3FUL3XQY2OYXRNTTSCWEHDANCNFSM4QEKNAQQ>
> .
>

TropeCode commented:
I don't see your latest edits. If they didn't get added automatically to this PR then I don't know what went wrong or how to fix it, but it if you open a new request we will lose the review comments.

Also, it would have been better if you responded to the review comments by replying to each thread (there are comment boxes on every one of them if you view the PR on the github site), so we can keep track of the context more easily.

Ven-edits commented:
Wow, sorry, I didn't realise there was this page on the Github site. I've been replying straight to emails!
I'll update those box threads with my comments.
I've just staged the file on my GitKraken and not sure what to do next. Do I push it, pull it, or drop it into the Silver Studio 1.39 workbranch?

Ven-edits commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -259,43 +259,43 @@ label hermione_intro_E1:
         pause.5
 
-        m "Yes, girl. It's all exactly as you say and I will.... take care of it all."
+        m "Yes, girl. It's all exactly as you say and I will... take care of it."
```
I removed the all here because it's doubling up from the one at the front of the sentence. He's already made 'all that she says' the topic so there's no need to repeat it at the end. You could remove the first all and leave the second one in though, but to my ears that doesn't sound as natural as leaving the first in.

Ven-edits commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -314,86 +314,85 @@ label hermione_intro_E1:
+    m "Well... I've read the books."
+    call sna_main("Books? What books?","snape_34")
+    m "Never mind. I'm a genie, remember? I know things."
+    call sna_main("Hmm... And yet you still need me to teach you magic?","snape_37")
```
I wasn't sure if you wanted italics or stars here. For now I've put it in stars since that's how all the other hmm's have been dealt with.

Ven-edits commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -538,66 +537,66 @@ label hermione_intro_E2:
+    m "My name's not--"
+    call sna_main("An esteemed wizard.","snape_08")
+    m "Well, alright, let it out."
+    call sna_main("How come one tiny... cunt is able to cause me so much grief?","snape_31")
```
Yeah, I pondered on this one a fair bit too. The comma isn't correct because it's splitting the sentence in two, and right off another pause at that. This is similar to the 'I mean, we' from earlier on so I'm wondering if what you're going for is stress on the word cunt which creates that slight pause after it as the voice readjusts. If so we'd put cunt in italics and remove the comma. For now I've put the comma back in, but it doesn't sit right with me.

Ven-edits commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -696,99 +695,99 @@ label hermione_intro_E3:
-    call her_main("I will know for sure when we get the results though...", "normal", "base", "base", "R", tears="soft")
-    call her_main("Yes, I'm sure everyone else failed as well...", "soft", "base", "base", "R")
+    call her_main("And the worst part is that I think I might be the only one who failed.", "angry", "base", "base", "mid", tears="soft")
+    call her_main("How would that make me look?", "angry", "base", "base", "mid", tears="soft")
```
The word 'this' sounds wrong to me. 'This' is more immediate than 'That', suggesting that the event has either happened and she's anticipating the repercussions, or that there's an object visually present that she's referring to in order to support her statement. Because she's only hypothesising that she may be the only one who failed, and it's not something that's happened or can be seen, the word 'that' feels more appropriate here so I've left it in for now. But it's your call.

Ven-edits commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -696,99 +695,99 @@ label hermione_intro_E3:
+    call her_main("Well, I'd better get going. We have another \"MRM\" meeting today.", "open", "closed", "angry", "mid")
+    her "I'll let you know about the new ideas we come up with."
+    m "I can hardly wait."
+    call her_main("Well, if there's nothing else, I have a studying schedule to keep.", "open", "closed", "base", "mid") # this is potentially doubling up her reason to leave. She's just said she has an MRM meeting to go to, so you could consider changing the line.
```
I've tweaked this slightly and brought the study schedule reference up behind the MRM meeting so she's selling her duties up but it's consolidated into a single sentence.
I can change it back, but from a player perspective this stood out as a genine error to me. The issue is that the two sentences have the same construction and are delivering the same information with different details, right down to the 'Well,' tagged on the front. So to me it looked like one was written to replace the other, but the original got left in by mistake and we ended up with them saying farewell to each other twice.

Ven-edits commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -861,83 +860,83 @@ label hermione_intro_E5:
                         pass
 
-            "\"Splendid! Starting today?\"":
+            "\"Splendid! Starting today?\"": # I'm unsure how these chat options connect just yet, but this is Genie speaking, right? He's asking her a question, but when Hermione comes back in she follows up by saying 'let me know when' rather than answering his question, so you might want to leave it at Splendid!
```
Fair enough. Though as a player I always assumed that the multiple choice options were what I was choosing for Genie to say/think/do. I actually selected this option (because I don't like to torment people...) so I was expecting Hermione to answer it. And she sort of does when she says 'Let me know when and I'll bring my books', but then she just walked out which left me wondering why Genie never bothered to give her the time to come back at. Obviously I can now see that it's because that final dialogue section is a one size fits all for every option.

Anyway, I've left it as it was for now.

Ven-edits commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -1090,101 +1089,101 @@ label hermione_intro_E6:
                     $ d_flag_01 = "desk"
                     $ d_flag_02 += 1
-                    her "*uhm*... alright..."
+                    her "*Uhm*... Okay."
```
I changed all of Hermione's alrights to okays to help differentiate her voice from Genie's a bit since they're together so often. Alright is used a lot, so it becomes rather noticeable when all the characters are saying it that it's a language quirk of the author. It's just a cheap trick, and at the end of the day they mean the exact same thing, so I can change this one back if you want.
I wouldn't read alright as coming across any more defeated than okay though. She's not really defeated in this situation, just hesitant, and contrary to the ellipsis zerg, the best way of showing that is with her wincing facial expression. But you could have her say something along the lines of "If you say so..." to show that she's somewhat reluctant about doing this.

Ven-edits commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -1263,38 +1262,38 @@ label hermione_intro_E6:
                 call her_main("..................", "annoyed", "squint", "angry", "mid")
-                call her_main("Yes, sir. I do need to get spanked.", "open", "closed", "angry", "mid")
-                m "Alright, that's good enough for now..."
+                call her_main("Yes, Sir. I deserve to be spanked.", "open", "closed", "angry", "mid")
```
I did miss the point here it seems. But to me, I only noticed that she changed up the word want with need, and not that Genie had used need in his earlier sentence. So this suggested to me that she wants to get spanked, which clearly isn't the case, so I changed it to deserve as that subtly satisfies his demand but isn't explicitly asking for it.
I've switched it back, but you could consider putting the italics stress on the word 'need' to show that she's playing on the word he just said and making the statement about the points for tired readers like me.

Ven-edits commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -1304,23 +1303,23 @@ label hermione_intro_E6:
 
-    call her_main("Thank you, professor. Thank you so much!", "smile", "happyCl", "base", "mid")
-    call her_main("Well... I suppose I'd better go now...", "base", "base", "base", "mid")
+    call her_main("Thank you, Professor Dumbledore. Thank you so much!", "smile", "happyCl", "base", "mid")
```
I changed it back, but I made this tweak because using his full name makes it more personal and emotional, and at the same time reminds the player that she's seeing him as Dumbledore and not Genie.

An interesting point about the names. To me, referring to somebody by their job title alone seems weird, as I don't tend to to think job titles should be used without the accompanying surname. But having done some research (Google) I see that in University they train people to do it that way to avoid the sexism issue of Miss vs Sir. I guess this is why people go to Uni! I should have probably have done that...

Ven-edits commented on game/_script_/_hermione_/events/_intro_.rpy:
```diff
@@ -1329,22 +1328,22 @@ label hermione_intro_E6:
-        call her_main("{size=-4}(I guess I'm a born actress...){/size}", "base", "happyCl", "base", "mid")
+        call her_main("{size=-4}(I'm a bad girl.){/size}", "angry", "base", "angry", "stare", xpos="base", flip=True, trans=d3)
+        call her_main("{size=-4}(I'm a very bad girl.){/size}", "base", "base", "angry", "stare")
+        call her_main("{size=-4}(Yes, it's easy to say meaningless things like that.){/size}", "smile", "happyCl", "base", "mid")
```
I've put it back, but I've moved easily up in front of say since it's a strange construction to have the adverb on the end of a pronoun which is what originally drew my eye to it. So it now reads "Yes, I can easily say things like that..."

TropeCode commented:
If you have your fork repository checked out with gitkraken, you can make changes to those files. Stages changes must be added by commit, that will finalise those changes on your computer, then you have to push the commit(s) to your online repository. When that's done the changes should appear in this pull request.

I recommend learning a bit about using git and/or gitkraken and getting comfortable with it, if you intend to make more pull requests in the future. In that case, knowing a bit about commits, branches and forks would be very useful.

Ven-edits commented:
A fair point. I think I've figured it out now. I've just been getting overwhelmed with all the coding lingo and buttons all over the place. The tool seems to be very powerful though.

Along with the Hermione Intro update I've also done a correction to the capitalisation of honorifics (those sirs and professors). I owe you an apology on that one. I thought you always capitalised when you used it as a replacement for a name, but apparently you don't do it when speaking to person in question directly, only when using it to refer to them with a third party. We live and learn.

David.

loafylemon commented:
I'm afraid we cannot merge this pull request, capitalisation of honorifics isn't strictly necessary because the game font displays text in capital letters only, and some of the proposed changes are changing the narrative a little too much. Also the amount of comments in this PR makes it impossible to keep track of the changes.

If you still wish to create a pull request with the dialogue changes, I would suggest contacting our writer @Johnny28, he can walk you through it.
